src/EnergyPlus/Humidifiers.cc

```diff
@@ -150,6 +149,7 @@ namespace Humidifiers {
                     state,
                     format("SimHumidifier: Invalid CompIndex passed={}, Number of Units={}, Entered Unit name={}", HumNum, NumHumidifiers, CompName));
             }
+            auto &CheckEquipName = state.dataHumidifiers->CheckEquipName;
```

Comment:
[src/EnergyPlus/Humidifiers.cc:132]:(style),[variableScope],The scope of the variable 'CheckEquipName' can be reduced.

Looks like I missed this one. Next pass.

[src/EnergyPlus/Humidifiers.cc:132]:(style),[constVariable],Variable 'CheckEquipName' can be declared as reference to const